### DISCUSSION TITLE
Device stream (IoT Hub)

### DISCUSSION THREAD

**Discussion by amitabhmodak** [Azure/azure-iot-sdk-c#2396] [General]
What is the roadmap for the IoT Hub Device Streams feature?

**Comment by ericwolz** [comment]
Currently there is no GA timeline for this feature.

**Reply by ghost** [reply]
Hi @ericwol-msft let us know when this feature will be available

**Reply by ericwolz** [reply]
Unfortunately, there still is no GA timeline for this feature.
